src/utils/excel_utils.js

```diff
@@ -50,33 +92,23 @@ export const loadProblemDataParallel = async (workbook, sheetNumber) => {
   }
 
   for (let g = 0; g < setNum; g++) {
-    setName = await sheet[`A${currentRow}`]['v'];
-    setType = await sheet[`B${currentRow}`]['v'];
+    setName = sheet[`A${currentRow}`]?.v || '';
+    setType = sheet[`B${currentRow}`]?.v || '';
     setNames.push(setName);
     setTypes.push(setType);
 
-    individualNum = await sheet[`D${currentRow}`]?.v;
+    individualNum = sheet[`D${currentRow}`]?.v || 0;
 
     for (let i = 0; i < individualNum; i++) {
-      let name = sheet[`A${currentRow + 1}`]?.v;
```

Comment:
Giữ cái cũ nhé:
      if (Object.is(name, undefined) || Object.is(name, null) || Object.is(name, '')) {
        name = `no_name_${i+1}`;
      }